index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio 👨‍💻 </title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+</head>
+<body>
+    
+    <header>
+        <h2>My logo</h2>
+       <nav>
+           <button type="button" class ="btn">&#9776;</button>
+       </nav>
+    </header>
+
+    <section class="headline-section">
+
+                <h1>I’m Abdifatah Yassen<br>
+                    Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you<br> build a product , feature or website Look<br>
+                         through some of my work and <br>experience! If you like what you see and<br> have a project you need
+                        coded, don’t <br>hestiate to contact me.</p>
```

Comment:
- [ ] Kindly remove the `<br />` tags in all of your texts here. By default, all texts are 100% responsive to their parent's width. The use of those tags within your texts is causing a bad experience for your texts. Please take them off and leave them as it is.

Reply:
Done